src/models/data.py

```diff
@@ -248,13 +371,26 @@ def __next__(self) -> Tuple[Tuple[Union[np.ndarray, torch.Tensor],
                     cur_max_idx = min(cur_max_idx + 1, self.max_idx)
 
                 out_x.append(arrays.x.historical)
+
+                # @GABI we don't want this do we ? it adds a column of constants
+                if self.experiment == 'nowcast':
+                    # add a constant vector to the X train_data and append
+                    constant_vector = np.ones(arrays.x.current.shape)[:, :, :1]
+                    arrays.x.current = np.concatenate(
```

Comment:
mypy error: mypy doesn't know `arrays.x.current` is not `None`. Same solution as above

Reply:
same as above